packages/client/hmi-client/src/components/model/tera-model.vue

```diff
@@ -87,8 +92,10 @@ const model = ref<Model | null>(null);
 const newName = ref('New Model');
 const isRenaming = ref(false);
 const isModelLoading = ref(false);
+const showSaveModal = ref(false);
 
 const isNaming = computed(() => isEmpty(props.assetId) || isRenaming.value);
+const isSaved = computed(() => isEqual(model.value, teraModelPartsRef.value?.transientModel));
```

Comment:
Just to note, these high-level equality checks can be a bit flaky because there are timestamps and versions numbers that might turn out to be always different. It seems like we are okay in this case but please be aware going foward.